### ISSUE TITLE
[question] does this solve the depth distortion?

### ISSUE THREAD

**Issue by warp1337** [ros-naoqi/extrinsic_calibration#1]
Does this calibration solve the distortion of depth camera? Why do you need to change the URDF?

**Comment by nlyubova** [comment]
Right, there is not need to change the URDF, it is better/easier to use the register_depth.launch file (I have just added it to launch folder and updated the README).

This code mainly helps to align the depth and color data. In order to improve the distortion, you would need to do intrinsic calibration for example as described in the tutorial: wiki.ros.org/pepper/Tutorials/Calibration
